VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -202,49 +170,24 @@ class VungleAdapter : PartnerAdapter {
                 .trim()
                 .takeIf { it.isNotEmpty() }
                 ?.let { appId ->
-                    Vungle.init(appId, context.applicationContext, object : InitCallback {
+                    VungleAds.init(context, appId, object : InitializationListener {
                         override fun onSuccess() {
-                            Plugin.addWrapperInfo(
-                                VungleApiClient.WrapperFramework.vunglehbs,
+                            VungleAds.setIntegrationName(
+                                VungleAds.WrapperFramework.vunglehbs,
                                 adapterVersion
                             )
 
-                            resumeOnce(
-                                Result.success(
```

Comment:
Oh, will reformat code, here. Since nothing was changed here. Unless we want to keep them as one-liners.